### ISSUE TITLE
Accessibility error: iframe has no title

### ISSUE THREAD

**Issue by limograf** [Shopify/buy-button-js#496]
https://www.w3.org/TR/WCAG20-TECHS/H64.html

Fails aXe core.

I think it's from here? https://github.com/limograf/buy-button-js/commit/a312f5d3850f16a6cd81ec988d0c3a3d3ea56e75

**Comment by codybrom** [comment]
+1 
Still failing accessibility in 2024 due to no iframe title
